website/siteConfig.js

```diff
@@ -43,18 +43,18 @@ const siteConfig = {
     // URL for editing docs, usage example: editUrl + 'en/doc1.md'.
     // If this field is omitted, there will be no "Edit this Doc" button
     // for each document.
-    editUrl: "https://github.com/mobxjs/mobx/edit/main/docs/",
+    editUrl: "https://github.com/mobxjs/ko.mobx.js.org/edit/main/docs/",
 
     // For top-level user or org sites, the organization is still the same.
     // e.g., for the https://JoelMarcey.github.io site, it would be set like...
     //   organizationName: 'JoelMarcey'
 
     // For no header links in the top nav bar -> headerLinks: [],
     headerLinks: [
-        { doc: "api", label: "API Reference" },
-        { href: "https://mobx.js.org", label: "English Doc" },
-        { doc: "backers-sponsors", label: "Sponsors" },
-        { href: "https://github.com/mobxjs/mobx", label: "GitHub" }
+        { doc: "api", label: "API 참고서" },
+        { href: "https://mobx.js.org", label: "영어 문서" },
+        { doc: "backers-sponsors", label: "후원자" },
+        { href: "https://github.com/mobxjs/ko.mobx.js.org", label: "GitHub" }
```

Comment:
Link point to mobx repository or translation repo, which is better ? If you are normal mobxjs user, click the github icon, you want to view mobxjs/mobx or mobxjs/ko.mobx.js.org ?  Give me a appropriate reasons and then I will merge this PR. Thank you.

Reply:
At first I thought mobxjs/en.mobx.js.org would be better, but after thinking about it again, I think mobxjs/mobx would be better. So I modified the url again.